permissions/PinQuery.permission

```diff
@@ -0,0 +1,13 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog =
+# x-sailjail-translation-key-description =
+# x-sailjail-description =
+# x-sailjail-translation-key-long-description =
+# x-sailjail-long-description =
```

Comment:
Since we're about to soon have the `x-sailjail-translation-catalog` to trigger the use of the translations (#66) shouldn't we put the keys and descriptions now when creating new permission files?

Reply:
I would argue that if the permission will not be needed separately and it doesn't have already defined texts, we shouldn't even need these lines.

Reply:
Ok. Do we need these translation content in each of the files or could these just be omitted from ones that are not going to be needed separately?

Reply:
We could omit them.

Reply:
Removed the extra cruft from this file. I think it's probably best to do the rest of these at once in a future task. Right now it would just create collisions with other PRs.